docs/usage/conventions.rst

```diff
@@ -5,7 +5,21 @@ This sections would introduce you conventions SCAutolib is using in sense of
 development, documentation, different approaches, and etc.
 
 1. Docstrings
-2. Packaging
-3. Unit test
-4. Architecture
-5. File structure
+----------------
+Each model, class, method, property or function has to have docstring.
+Description can be short, but has to be clear.
+For methods and function, docstring has have specification of arguments with data type, return value with data type and possible exceptions.
+
+2. Unit test
+--------------
+All code should be covered by unit test unless there is specific reason preventing it.
+Especially, unit tests are required for functionality can be tested without changing the system state or do not require any special setup.
```

Comment:
**that** are required?  or just required without **are** . This sentence is difficult to understand.

Reply:
"Especially, unit tests are required for functionality **that** can be tested ..."